acme/LICENSE.txt

```diff
@@ -0,0 +1,190 @@
+   Copyright 2015 Internet Security Research Group
```

Comment:
As discussed in IRC: no, we cannot symlink, because it breaks Python packaging (c.f #838).